### DISCUSSION TITLE
Help for adding channels

### DISCUSSION THREAD

**Discussion by allanf181** [iptv-org/community#208] [Show and tell]
https://i.mjh.nz
http://www.apsattv.com/streams.html

These two sites have public channel lists that are updated daily. It would be a great way to add them and keep them up to date.

**Comment by BellezaEmporium** [comment]
If @matthuisman is OK with the idea, we could import his mjh.nz channels, although I believe we do have some channels from his server located in the nz channels.

**Reply by matthuisman** [reply]
fine with me. just link to my site or github if possible :)
and i dont want to be providing support for streams not being used in the way I intend :)

**Comment by allanf181** [comment]
a bot fetching all.json and splitting between the lists seems to be the best way, running once a week should be enough
